openlibrary/templates/merge_request_table/table_row.html

```diff
@@ -1,94 +1,174 @@
-$def with (request, username, can_merge)
-$# request : dict : Dict containing information about a single merge request.
+$def with (request, username, can_merge, is_deletion_mode=False)
```

Comment:
```suggestion
$def with (request, username, can_merge)
```
Remove unnecessary parameter.